### PR TITLE
Add script for restoring to staging (DBAI-46)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR aims to resolve DBAI-46. It is still a draft.

To Do
- [x] Add basic implementation
- [x] Reconcile with `canister` PR
- [x] Test job

### CODE REVIEW

**Review by jayamala17 — APPROVED** [review]

File: restore_from_aptrust.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+config = Config::ConfigService.from_file(File.join(".", "config", "config.yml"))
+SemanticLogger.default_level = config.settings.log_level
+
+class RestoreError < StandardError
+end
+
+class RestoreJob
+  include SemanticLogger::Loggable
+
+  def initialize(config:, base_path: "umich.edu")
```
Comment (jayamala17) [inline restore_from_aptrust.rb:16]: can we store base_path in the config instead of directly assigning ?

Comment (ssciolla) [reply restore_from_aptrust.rb:16]: Sure, something like that, I just put it as a default for now as part of my initial pass. I may have it somewhere else in the code, I'll have to look.

**Review by jayamala17 — APPROVED** [review]

File: restore_from_aptrust.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
+require "semantic_logger"
+
+require_relative "lib/config"
+require_relative "lib/remote_client"
+
+SemanticLogger.add_appender(io: $stderr, formatter: :color)
```
Comment (jayamala17) [inline restore_from_aptrust.rb:6]: logger and config can be used from require_relative "services", just this change  on this file.

Comment (ssciolla) [reply restore_from_aptrust.rb:6]: Will do.

Comment (ssciolla) [reply restore_from_aptrust.rb]: Done. https://github.com/mlibrary/bag-courier/pull/40/commits/e45f4cc6c6a0b6661789be43903fee856dc533e7

**Review by jayamala17 — APPROVED** [review]

**Pull request merged**